app/assets/javascripts/events.js

```diff
@@ -1,2 +1,30 @@
 // Place all the behaviors and hooks related to the matching controller here.
 // All this logic will automatically be available in application.js.
+
+jQuery(function() {
+
+    $('#send-emails-modal').on('show.bs.modal', function (event) {
```

Comment:
It is bootstrap component http://getbootstrap.com/javascript/#modals. But this will be replaced from another view next sprint.